test-data/unit/check-namedtuple.test

```diff
@@ -185,3 +200,104 @@ class C(B): pass
 B(1).b
 C(2).b
 [builtins fixtures/property.pyi]
+
+[case testNamedTupleAsDict]
+from collections import namedtuple, OrderedDict
+
+X = namedtuple('X', ['x', 'y'])
+x = None  # type: X
+a = None  # type: OrderedDict
+a = x._asdict()
```

Comment:
I'd suggest using `reveal_type` instead of assignment to test the return type of `x._asdict()`, as it'll catch things unexpected `Any` types.